docs/developer-guide/maven-indexer.md

```diff
@@ -43,65 +43,111 @@ except a record for the POM.
 Maven indexer also may or may not store a `.pom` file as an artifact. However, firstly it tries to find matching _real_ 
 artifact file in the file system, switching over to indexing that, instead of the `.pom` file.
 
-### What's not indexable
+### What's not indexable ?
 
 The following file types are not indexable:
 
 * `maven-metadata.xml` files
 * `.properties` files
 * checksum and signature files `.asc`, `.md5`, `.sha1`
 
+### What Are Packed Indexes?
+
+Packed indexes are either a complete compressed index, or a compressed subset of data which can be applied to an 
+existing index incrementally.
+
+## What's the goal of packed indexes ?
+
+Packed indexes are used for transferring indexes from the remote to the proxy/tool. 
+
 ## What Is The Maven Indexer Used For In The Strongbox Project?
 
 The Maven Indexer is used for integration with IDE-s.
 
-The Maven indexes produced by most public repository managers (such as Maven Central), are usually rebuilt once a week, 
-as it can take quite a while to scan large repositories with countless small artifacts. Hence, these indexes have proven
- to not be quite as up-to-date, as the real server's contents. For this reason, we are using OrientDB to keep more 
- accurate information.
+## How Does The Maven Indexer Work In Strongbox ?
+
+Strongbox allows you to download packed repository Maven Index. Every maven repository with indexing enabled serves the packed Maven Index. 
+Based on the repository type, the index is prepared as follows:
+
+* [Hosted](../knowledge-base/repositories.md#hosted) repositories:
```

Comment:
You should use an alias links when the link appears multiple times in the same document to make it easier for when the link needs to be changed.


```suggestion

* [Hosted][hosted-repositories-link] repositories:
* [Group][group-repositories-link] repositories:


--- somewhere at the very bottom ---

[hosted-repositories-link]: ../knowledge-base/repositories.md#hosted
[group-repositories-link]: ../knowledge-base/repositories.md#group
-- and so on
```

Reply:
Great to know that, thanks :)